### ISSUE TITLE
ADJUST_RESIZE validation

### ISSUE THREAD

**Issue by alzhuravlev** [yshrsmz/KeyboardVisibilityEvent#38]
Hi,

thank you for such a useful lib!

I am not sure that code below correctly check that activity has `WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE`

may be 
```java
WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE & softInputMethod  != 0
```
?
in my case  softInputMethod == 275 which tells us that activity is ADJUST_RESIZE

KeyboardVisibilityEvent.java
```java
        int softInputMethod = activity.getWindow().getAttributes().softInputMode;
        if (WindowManager.LayoutParams.SOFT_INPUT_ADJUST_UNSPECIFIED != softInputMethod &&
                (WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE > softInputMethod ||
                        WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN <= softInputMethod)) {
            throw new IllegalArgumentException("Parameter:activity window SoftInputMethod is not ADJUST_RESIZE");
        }
```

thisnk you

**Comment by yshrsmz** [comment]
Indeed this looks more correct approach to a bit flag.
Would you like to submit a PR?

**Comment by yshrsmz** [comment]
this may also fix #37 ?

**Comment by jemshit** [comment]
> this may also fix #37 ?

Delaying did not fix

**Comment by yshrsmz** [comment]
I mean checking the flag this way may fix #37.

**Comment by alzhuravlev** [comment]
Hi, Do you plan to merge my pull request? I experience crashes in my prod env right now(

**Comment by yshrsmz** [comment]
give me few hours, I'm reviewing now.

**Comment by yshrsmz** [comment]
I've just released v2.3.0.
Thanks for your contribution!
